src/dynamo_requests.jl

```diff
@@ -63,11 +63,11 @@ function signature_version_4(env, service, method, host, action, payload)
     authorization_header = algorithm * " " * "Credential=" * env.aws_id * "/" * credential_scope * ", " *
         "SignedHeaders=" * signed_headers * ", " * "Signature=" * signature
 
-    headers = [("Authorization", authorization_header),
-               ("X-Amz-Date", amzdate),
-               ("Host", host),
-               ("Content-Type", "application/x-amz-json-1.0"),
-               ("X-Amz-Target", "DynamoDB_20120810.$action")]
+    headers = Dict("Authorization" => authorization_header,
+               "X-Amz-Date" => amzdate,
+               "Host" => host,
+               "Content-Type" => "application/x-amz-json-1.0",
+               "X-Amz-Target" => "DynamoDB_20120810.$action")
```

Comment:
(I know this somewhat off topic for this request, but still:) a dict here is soo much better